Tools/PlatformHaiku.cmake

```diff
@@ -1,5 +1,8 @@
 add_subdirectory(TestRunnerShared)
-add_subdirectory(DumpRenderTree)
-add_subdirectory(HaikuLauncher)
 add_subdirectory(ImageDiff)
-add_subdirectory(MiniBrowser/haiku)
\ No newline at end of file
+add_subdirectory(MiniBrowser/haiku)
```

Comment:
For symetry with other platforms this should be guarded by ENABLE_MINIBROWSER, which in turn should be enabled in Source/cmake/OptionsHaiku.cmake as it's done for PlayStation:

    WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_MINIBROWSER PUBLIC ${ENABLE_WEBKIT})

This way it will still be possible to build WebKitLegacy only (that doesn't use MiniBrowser).